devito/interfaces.py

```diff
@@ -127,7 +132,10 @@ def set_initializer(self, lambda_initializer):
         self.initializer = lambda_initializer
 
     def _allocate_memory(self):
-        self._data = aligned(np.zeros(self.shape, self.dtype, order='C'), alignment=64)
+        if self.memmap:
+            self._data = aligned(np.memmap(filename=(self.f), dtype=self.dtype, mode='w+', shape=tuple(self.shape), order='C'), alignment=64)
```

Comment:
I am not sure about this, I just copy paste from master. Would you like to have it removed?